File: examples/hello.js
// Example JavaScript/Node.js script
console.log("🚀 Hello from JavaScript!");
console.log("=".repeat(30));

// Variables and operations
const name = "AI-IDE";
const version = process.version;
console.log(`Running on Node.js ${version}`);
console.log(`Welcome to ${name}!`);

// Array operations
const numbers = [1, 2, 3, 4, 5];
const squares = numbers.map(n => n * n);
console.log(`Numbers: ${numbers}`);
console.log(`Squares: ${squares}`);

// Object/JSON example
const project = {
    name: "AI-IDE",
    language: "JavaScript",
    runtime: "Node.js",
    features: ["execution", "terminal", "real-time"]
};
console.log("Project info:", JSON.stringify(project, null, 2));

// Async example
setTimeout(() => {
    console.log("\n✅ JavaScript execution complete!");
}, 100);

File: examples/hello.py
#!/usr/bin/env python3
"""
Example Python script demonstrating various features
"""

print("🐍 Hello from Python!")
print("=" * 30)

# Math operations
result = 10 + 5 * 2
print(f"Math: 10 + 5 * 2 = {result}")

# String operations
name = "AI-IDE"
print(f"Welcome to {name}!")

# List operations
numbers = [1, 2, 3, 4, 5]
squares = [n**2 for n in numbers]
print(f"Squares: {squares}")

# JSON example
import json
data = {
    "project": "AI-IDE",
    "language": "Python",
    "version": "3.11",
    "features": ["execution", "terminal", "file management"]
}
print(f"JSON: {json.dumps(data, indent=2)}")

print("\n✅ Python execution complete!")